response_operations_ui/views/passwords.py

```diff
@@ -127,8 +127,14 @@ def resend_password_email_expired_token(token):
 def send_password_change_email(email):
     url_safe_serializer = URLSafeSerializer(app.config['SECRET_KEY'])
 
-    first_name = uaa_controller.get_first_name_by_email(email)
-    if first_name != "":
+    response = uaa_controller.get_user_by_email(email)
+    if response.status_code != 200:
+        logger.error('Error retrieving user from UAA', status_code=response.status_code,
+                     email=url_safe_serializer.dumps(email))
+        return render_template('forgot-password-error.html')
+
+    if response.json()['totalResults'] > 0:
```

Comment:
What happens if totalResults isn't in the dict?  Currently it'd throw a `TypeError`, log a `A generic exception happened` line, then show the error-500 page.

Reply:
It's not possible for this key to be missing on a `200` response from the UAA API

Reply:
Oki dokes.  The stacktrace would let us know what happened if that element magically disappeared anyway